### ISSUE TITLE
update stack view

### ISSUE THREAD

**Issue by kirinyaga** [kierenj/0x10c-DevKit#195]
Content of stack is not updated after each step. As I extensively use the top of the stack to store temporary working variables, I have to click on the checkbox after every step to see the updated values. 
Stack is also used to store local variables if your function needs to be reentrant, which is a good idea anyway.

**Comment by kierenj** [comment]
It should be updated after each step - it is for me. Can you try restarting Devkit and trying again?  There is a bug where if you select 10khz mode with the vector display plugin loaded, things go very wrong from that point, which might explain it. Let me know? :)

**Comment by kirinyaga** [comment]
To be clear : the stack is properly updated on push, pop & jsr. It is not however (confirmed on three win7 installs, including one vm) with this code :
set x, 4
set a, 7
set push, x
shr peek, 1 ; <= stack view not updated
mul x, a
add x, pop   ; x=ax + x/2

**Comment by kirinyaga** [comment]
yeah, I read my first message again : it was not clear at all, sorry. The problem is when you change the content of already pushed values (that is the temporary working variables or non-static function local variables).
